packages/perspective-webpack-plugin/src/js/psp-worker-loader.js

```diff
@@ -55,17 +54,15 @@ exports.default = function loader(content) {
                 .replace(/(cjs\/js)/, "build");
         }
         content = fs.readFileSync(inputPath).toString();
-        if (!options.inline) {
```

Comment:
This causes duplicate hashed webworker assets to be output in the Jupyterlab plugin on every build

<img width="946" alt="screen shot 2019-02-01 at 10 08 20 pm" src="https://user-images.githubusercontent.com/60666/52159133-0134b400-266e-11e9-8aeb-196b48ba19c6.png">